src/api/queries/getMetricQueries.js

```diff
@@ -0,0 +1,65 @@
+function currencyNameHandler (metric, options) {
+  var currencies = ['usd']
+  var newOrderMetrics = {
+    'mvrv_long_short_diff': 'mvrv{currency}_long_short_diff'
+  }
+  var metricName = metric
+
+  if ('currency' in options) {
+    if (metric in newOrderMetrics) {
+      if (typeof options.currency === 'undefined') {
+        metricName = newOrderMetrics[metric].replace('{currency}', '_usd')
+      } else if (currencies.indexOf(options.currency) !== -1) {
+        metricName = newOrderMetrics.metric.replace('{currency}', metric.currency)
+      } else {
+        throw new Error('This is not a valid currency')
+      }
+    } else if (!(metricName in newOrderMetrics)) {
```

Comment:
why `metricName` is used here and `metric` is used in the previous `in newOrderMetrics` check?

Reply:
using `var` is very confusing. As it's not scoped

Reply:
For the first question, you're right, I don't really remember why I did so, I guess it's a mistake. The problem with var is that in this file I cannot use anything else, because it should be in ECMA 5 and let is introduced in ECMA 6.

Reply:
@IvanIvanoff `src` is using the old JS, `var` should be used